src/HornedBeast.js

```diff
@@ -1,15 +1,38 @@
 import React from 'react';
+import Image from 'react-bootstrap/Image'
 
 class HornedBeast extends React.Component {
-  render() {
-    return (
-      <>
-        <h2>{this.props.title}</h2>
-        <img src={this.props.img_url} alt='' title='Billy'/>
-        <p>{this.props.description}</p>
-      </>
-    )
+  constructor(props) {
+    super(props);
+    this.state = {
+      "status":"Click If You Like It!"
+    }
   }
-};
 
-export default HornedBeast;
\ No newline at end of file
+  handleClick = () => {
+    if (this.state.status === "Click If You Like It!") {
+      this.setState({
+        status: "💜 1"
+      })
+      } else {
+        let temp = parseInt(this.state.status.replace(/[^0-9]/g, ''))+1;
+        this.setState({
+          status: "💜" +temp
+        })
+    
+    };
+  }
+
+    render() {
+      return (
+        <div onClick={this.handleClick}>
+        <Image src= {this.props.image_url} alt="coming soon" rounded fluid/>
+        <h4>
+          {this.state.status}
+        </h4>
+      </div>
```

Comment:
In each HornedBeast component you want to have the Title, description, etc. An easy way to keep all this information organized is with Cards from React Bootstrap. Check out how Cards work and think about implementing them here.